frontend/resources/translations/en.json

```diff
@@ -253,7 +253,8 @@
     "moreButtonToolTip": "Remove, rename, and more...",
     "addPageTooltip": "Quickly add a page inside",
     "defaultNewPageName": "Untitled",
-    "renameDialog": "Rename"
+    "renameDialog": "Rename",
+    "copy": "Copy"
```

Comment:
Please give a more descriptive name to this translation token. Simply "copy" is misleading